api/handlers/stripe.js

```diff
@@ -3,28 +3,44 @@ const stripeAPI = require('stripe');
 const {
     STRIPE
 } = require('../config/credentials')
+const db = require('../models')
 
 const stripe = module.exports = (() => {
+    const stripeClient = stripeAPI(STRIPE.SECRET)
 
-    const requestPaymentIntent = (params) => {
-        const paymentIntent = new Promise((resolve, reject) => {
-            const stripeClient = stripeAPI(STRIPE.API_KEY)
-            stripeClient.paymentIntents.create({
-                amount: 1000,
-                currency: 'usd',
-                payment_method_types: ['card'],
-                receipt_email: 'test@example.com',
+    const createClient = async (params) => {
+        const client = await findClientWithEmail(params.createFields)
+        if (!client.external_uuid) {
+            return await stripeClient.customers.create({
+                email: params.createFields.email,
+                name: params.createFields.name,
             })
-                .then(response => {
-                    resolve(response)
-                })
-                .catch(error => reject(new Error(error)))
+        }
+    }
+
+    const findClientWithEmail = async (params) => {
+        return await db.models.Client.findOne({
+            where: {
+                email: params.email
+            }
         })
-        return paymentIntent
+    }
```

Comment:
For a better separation of concerns between API module functionality and Stripe-specific API handling, let's move this logic + function to a new file called `/modules/clientManagement`